### DISCUSSION TITLE
Having a configuration option to restart the program after the computer comes back from Hibernate or Sleep

### DISCUSSION THREAD

**Discussion by ghost** [wwmm/easyeffects#2849] [Ideas]
My program freezes when the computer enters Hibernation or Sleep mode. I believe a solution would be to add a configuration option to automatically restart the program upon the computer's return from these modes. This would help bypass the issue and ensure a more stable experience.

**Comment by ghost** [comment]
Good evening, apparently this error is only caused in the flatpak version.

**Reply by wwmm** [reply]
>Good evening, apparently this error is only caused in the flatpak version.

Weird...

**Comment by wwmm** [comment]
As EasyEffects is just an ordinary app that pretends to be a service it can´t really restart itself. The most that could be done is writing a separated service that could try to do this. But the thing is that since when we were still using Pulseaudio this sleep/hybernation bug has been a mystery. It depends on the hardware and the reason why some hardware have it was never found.
